docs/apidocs/ansatz_generation.rst

```diff
@@ -1,10 +1,20 @@
-.. _ansatz_generation:
-
-#################
-Ansatz Generation
-#################
+####################################################################
+Ansatz Generation (:mod:`qiskit_addon_aqc_tensor.ansatz_generation`)
+####################################################################
 
 .. automodule:: qiskit_addon_aqc_tensor.ansatz_generation
    :no-members:
    :no-inherited-members:
-   :no-special-members:
\ No newline at end of file
+   :no-special-members:
+
+.. currentmodule:: qiskit_addon_aqc_tensor.ansatz_generation
+
+.. autosummary::
```

Comment:
I didn't inline because we show the inherited `Gate` methods, which are long although I think very useful to users

Reply:
> we show the inherited `Gate` methods, which are long although I think very useful to users

One one hand it makes sense to me to do this.  On the other, I expect that most of them will be rarely used, and the information from `Gate` seems to dominate all other information, almost crowding out what is truly useful.  What if each of them instead just said somehow that the class implements all the methods of `Gate` (link to it).